src/ppx/grfx/grfx_pipeline.cpp

```diff
@@ -12,16 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "ppx/config.h"
 #include "ppx/grfx/grfx_device.h"
 #include "ppx/grfx/grfx_pipeline.h"
 #include "ppx/grfx/grfx_descriptor.h"
+#include "ppx/grfx/grfx_enums.h"
 
 namespace ppx {
 namespace grfx {
 
 // -------------------------------------------------------------------------------------------------
 // BlendAttachmentState
 // -------------------------------------------------------------------------------------------------
+grfx::BlendAttachmentState BlendAttachmentState::BlendModeNone()
+{
+    grfx::BlendAttachmentState state = {};
```

Comment:
it would be better to be explicit about the blend mode with `state.blendEnable = false;` so that it is clear that we are not doing any blending